src/main/resources/db/migration/V2_17__add_actioned_by_to_visit.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE visit ADD created_by VARCHAR(30);
```

Comment:
is 30 length the max we will deal with, it's fairy big I suppose

Reply:
thanks, updated to 60 as discussed